src/test/ui/macros/macro-non-lifetime.stderr

```diff
@@ -6,6 +6,12 @@ LL | macro_rules! m { ($x:lifetime) => { } }
 ...
 LL |     m!(a);
    |        ^ no rules expected this token in macro call
+   |
+note: while trying to match `meta-variable $x:lifetime`
```

Comment:
Should the word "meta-variable" be outside the code-quotes?

Reply:
yeah the quoting is quite bad i need to fix it